sormas-api/src/main/java/de/symeda/sormas/api/hospitalization/HospitalizationDto.java

```diff
@@ -60,6 +62,8 @@
 	private YesNoUnknown intensiveCareUnit;
 	private Date intensiveCareUnitStart;
 	private Date intensiveCareUnitEnd;
+	private HospitalizationReasonType reasonForHospitalization;
```

Comment:
I'd prefer to call the field "hospitalizationReason" - more consistent with the enum name and also shorter. The caption can stay the way it is of course.